resources/i18n/qqq.json

```diff
@@ -50,5 +50,129 @@
+	"save-file-as-window-title":"Represents the action of saving a file with its name based on the window title or a chosen name.",
+	"download-finished-message":"Text displayed when the download action has been completed",
+	"tools":"{{Identical|Tools}}",
+	"window-title":"The title text displayed when the library browser is present",
+	"browse-directory":"Represents the action of browsing the local file system for a directory.",
+	"settings":"{{Identical|Settings}}",
+	"forward":"{{Identical|Forward}}",
+	"search-files":"Represents the action of searching online or locally for Zim files.",
+	"content-type":"Describes the type of contents available on a Zim file, for example, text or pictures.",
+	"reset-sort":"Represents the action of resetting the sorting of the items to default order.",
+	"open":"{{Identical|Open}}",
+	"download":"{{Identical|Download}}",
+	"resume":"\"Resume\" here refers to continuing a task.",
+	"port-for-local-kiwix-server-setting":"Describes the port of the computer of which the local Kiwix server will be hosted on.",
+	"zoom-level-setting":"Describes the default zoom level of articles displayed on this desktop application.",
+	"download-directory-setting":"Describes the directory in the file system where Zim files will be downloaded to.",
+	"monitor-directory-setting":"Describes the directory in the file system where we will monitor for Zim file changes.",
+	"monitor-directory-default":"Describes the default directory in the file system where we will monitor for Zim file changes.",
+	"reset":"{{Identical|reset}}",
+	"clear":"\"Clear\" here refers to emptying something.",
+	"browse":"{{Identical|Browse}}",
+	"about-kiwix-desktop-title":"Describes the title of this desktop application",
+	"about-kiwix-desktop-description":"Brief description of this desktop applciation.",
+	"about-learn-more-about-kiwix":"Hyperlink text directing to more details about Kiwix.",
+	"about-release-title":"Title text for the section on the release stages of this desktop application.",
+	"about-source-code":"Describes the licensing of the source code used for this desktop application.",
+	"about-version":"Describes the release version of this desktop application.",
+	"about-reporting-problem-title":"Title text for the section for how to report a problem for this desktop applciation.",
+	"about-report-issue":"Description of instructions on how to report an issue.",
+	"about-report-issue-2":"Description of instructions on how to report an issue.",
+	"about-libraries-title":"Title text for the section on what libraries are used in this desktop applciation.",
+	"kiwix-server-running-message":"Text displayed when kiwix server has started running.",
+	"kiwix-server-description":"Description text about capabilities of Kiwix server",
+	"fullscreen-notification":"Notification Text displayed when the desktop applciation is set to fullscreen.",
+	"online-files":"The Zim files that can be found online.",
```

Comment:
files → flies

Reply:
This one is actually not a typo. I can feel your eye strain rn.

Reply:
:+1: This was actually a test that you don't take my feedback blindly